package.json

```diff
@@ -22,7 +22,7 @@
     "prepush": "yarn test && yarn test:routes"
   },
   "dependencies": {
-    "@hmcts/cmc-common-frontend": "^1.4.11",
+    "@hmcts/cmc-common-frontend": "^1.5.11",
```

Comment:
Shouldn't be new version 1.5.0?

Format A.B.C

If you update B, C should be zero'd?
If we update A, B and C = 0?

@geek-soft

Reply:
it should have been but i did it by mistake and changing it makes no sense at the moment. when we next make a change I will update it to 1.6.0

Reply:
ok

Reply:
actually can you please make another `npm release` for common fe and then update this dependency.